web/src/components/zonelist.js

```diff
@@ -93,6 +95,7 @@ const ZoneList = ({
 
   const enterZone = (zone) => {
     dispatchApplication('mapViewport', getCenteredZoneViewport(zone));
+    trackEvent('zoneEntered', {zoneEntered: zone.countryCode});
```

Comment:
Hmm, I'd suggest either using the same event for selecting a zone (and then have an attribute for determining the source), or having two events clearly named (e.g. "zoneSelectedFromRanking", "zoneSelectedFromMap"). I think the first option is best, but don't know how well that works in Plausible :)

Reply:
Ok so how about

trackEvent('zoneSelected', {zoneSelected: zone.countryCode, zoneSelectedSource: "ranking"});

Note that the way our event system is setup, we always send all attributes in the context alongside, so all events in plausible has like 10-15 attributes.